ui/DesignSystem/Component/ProjectListItem.svelte

```diff
@@ -0,0 +1,32 @@
+<script>
```

Comment:
```suggestion
<script lang="ts">
```
All new components should be typescripted.